src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_horizontal_layout.html.twig

```diff
@@ -0,0 +1,72 @@
+{% use "bootstrap_4_layout.html.twig" %}
+
+{% block form_start -%}
+    {% set attr = attr|merge({class: attr.class|default('')|trim}) %}
+    {{- parent() -}}
+{%- endblock form_start %}
+
+{# Labels #}
+
+{% block form_label -%}
+{% spaceless %}
```

Comment:
We don't use ``spaceless`` anymore in form themes. this has a perf impact as it performs a regex replacement on the rendered string. You should use whitespace control to remove the unnecessary spaces during the template compilation instead